node/text-to-speech-with-huggingface/src/main.js

```diff
@@ -0,0 +1,47 @@
+import { HfInference } from '@huggingface/inference';
+import { getStaticFile, throwIfMissing } from './utils.js';
+
+export default async ({ req, res }) => {
+  throwIfMissing(process.env, ['HUGGINGFACE_ACCESS_TOKEN']);
+
+  if (req.method === 'GET') {
+    return res.send(getStaticFile('index.html'), 200, {
+      'Content-Type': 'text/html; charset=utf-8',
+    });
+  }
+
+  if (req.method !== 'POST') {
+    return res.json({ ok: false, error: 'Method not allowed.' }, 405);
+  }
+
+  if (!req.body.source || typeof req.body.source !== 'string') {
+    return res.json(
+      { ok: false, error: 'Missing required field `source`' },
+      400
+    );
+  }
+
+  const hf = new HfInference(process.env.HUGGINGFACE_ACCESS_TOKEN);
```

Comment:
Is this ACCESS_TOKEN or API_TOKEN? In the readme for this function I see API_TOKEN. 
Please search through the other examples as well to make sure we use the same variable

Reply:
ditto